VERSION

```diff
@@ -1 +1 @@
-1.0.2
+1.0.3
```

Comment:
I'm not sure if this is only a patch version bump, since we are adding functionality in a backwards compatible manner, I might lean towards `1.1.0`

Reply:
It's pretty minor, but sure.